File: core/src/si/feri/slidegame/utils/Pair.java
package si.feri.slidegame.utils;

public class Pair<A, B> {
    public A fst;
    public B snd;

    public Pair(A fst, B snd) {
        this.fst = fst;
        this.snd = snd;
    }

    public static <A, B> Pair<A, B> of(A fst, B snd) {
        return new Pair(fst, snd);
    }
}

File: core/src/si/feri/rrizemljevid/utils/Pair.kt
package si.feri.rrizemljevid.utils

class Pair<A, B>(var fst: A, var snd: B) {
    companion object {
        fun <A, B> of(fst: A, snd: B): Pair<A, B> {
            return Pair(fst, snd)
        }
    }
}